doc/source/examples/analysis/reduced_dimensions/diffusion_map.ipynb

```diff
@@ -137,7 +137,8 @@
     "ExecuteTime": {
      "end_time": "2023-12-01T15:45:48.442257804Z",
      "start_time": "2023-12-01T15:45:48.414223207Z"
-    }
+    },
+    "tags": ["nbval-ignore-output"]
```

Comment:
rather than keep updating the notebooks every time the behaviour changes, it's probably better to just do this?

Reply:
Yeah, I think that's a good thing in this case.